functions/go/set-enforcement-action/README.md

```diff
@@ -0,0 +1,42 @@
+# set-enforcement-action
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Applies the supplied enforcement action on [policy constraints](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller#constraints) within a package.
+
+<!--mdtogo-->
+
+Policy Controller [constraint objects](https://cloud.google.com/anthos-config-management/docs/how-to/creating-constraints) enable you to enforce policies for your Kubernetes clusters.
+This function provides a quick way for users to set the [enforcementAction](https://cloud.google.com/anthos-config-management/docs/how-to/auditing-constraints#types_of_enforcement_actions)
```

Comment:
> [enforcementAction](...

If referring a field, please use a full path and backquote it. i.e. 
```
[`spec.enforcementAction`](...
```

Reply:
Done.